stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/FailoverContext.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.net.http.websocket.client;
+
+import java.util.List;
+
+/**
+ * Represents a failover client connector config.
+ *
+ * @since 1.2.0
+ */
+public class FailoverContext {
+
+    private boolean finishedFailover = false;
+    private int currentIndex = 0;
+    private int failoverInterval = 0;
+    private boolean firstConnectionMadeSuccessfully = false;
+    private List<String> targetUrls = null;
+    private int initialIndex = 0;
+
+    /**
+     * Get the value of completed failover.
+     *
+     * @return finishedFailover
+     */
+    public boolean isFailoverFinished() {
+        return finishedFailover;
+    }
+
+    /**
+     * Assign the finishedFailover of the FailoverContext to the variable finishedFailover.
+     *
+     * @param finishedFailover - if it is true, do reconnect in the target URL.
+     */
+    public void setFailoverFinished(boolean finishedFailover) {
+        this.finishedFailover = finishedFailover;
+    }
+
+    /**
+     * Get the index.
```

Comment:
```suggestion
     * Gets the index.
```
Please check all occurrences if we have used the singular form to preserve consistency.